app/views/activities/_form.html.slim

```diff
@@ -1,3 +1,4 @@
+- @activity = Activity.new
```

Comment:
Is that tolerable? Or even ok?
This happened before we figured out how to trigger AJAX :)